Containers/mastercontainer/internal.Caddyfile

```diff
@@ -0,0 +1,29 @@
+{
+	auto_https off
+
+	storage file_system {
+		root /mnt/docker-aio-config/caddy/
+	}
+
+	log {
+		level ERROR
+	}
+
+	servers {
+		protocols h1 h2
+	}
+
+	skip_install_trust
```

Comment:
when https://github.com/caddyserver/caddy/issues/7311 is fixed and we switch back to one caddy this option also needs to be added to the single Caddyfile then